scripts/deploy.js

```
@@ -13,7 +13,14 @@ if (fs.existsSync(distDir)) fs.rmSync(distDir, { recursive: true, force: true })
 console.log('\n▶ Building web app...');
 execSync('npx expo export --platform web', { cwd: root, stdio: 'inherit' });
 
-// 2. Fix Ionicons for web
+// 2. Copy web/manifest.json into dist so PWA standalone mode works
+const webManifest = path.join(root, 'web', 'manifest.json');
+if (fs.existsSync(webManifest)) {
+  fs.copyFileSync(webManifest, path.join(distDir, 'manifest.json'));
+  console.log('✓ manifest.json copied to dist');
```

ESLint

[error] 'console' is not defined.

(no-undef)